resources/lang/dk_DA.json

```
@@ -0,0 +1,33 @@
+{
+   "noRecordsFound":         "Ingen felter fundet",
+   "showingPage":            "Viser side {0} af {1}",
+   "show":                   "Vis ",
+   "search":                 "Søg",
+   "entries":                "indgange",
+   "columnName":             "Søjlenavn",
+   "insertANewColumnBefore": "Indsæt ny søjle før",
+   "insertANewColumnAfter":  "Indsæt ny søjle efter",
+   "deleteSelectedColumns":  "Slet valgte søjler",
+   "renameThisColumn":       "Omdøb denne søjle",

```

Translating "column" to "kolonne" would be more accurate.

```suggestion
   "columnName":             "Kolonnenavn",
   "insertANewColumnBefore": "Indsæt ny kolonne før",
   "insertANewColumnAfter":  "Indsæt ny kolonne efter",
   "deleteSelectedColumns":  "Slet valgte kolonner",
   "renameThisColumn":       "Omdøb denne kolonne",

```